doc/howto/instances_create.md

```diff
@@ -60,6 +60,11 @@ To launch a virtual machine with a Ubuntu 22.04 image from the `images` server u
 
     lxc launch images:ubuntu/22.04 ubuntu-vm --vm
 
+```{tip}
+When creating a virtual machine, use the `cloud` variant of an image (you can identify it by checking for the `cloud`-tag in the image alias).
```

Comment:
Why is this tip specific to VMs? Also this feels like it should be a separate commit.

The cloud variants are available for both containers and VMs. 
One thing that might be worth pointing out (if not already done so) is that using the cloud-init variant will allow for root filesystem resizing to the configured root disk size, whereas otherwise it needs to be done manually.

Reply:
Also the `cloud` variant is only really relevant to the `images:` remote.

Reply:
This note comes from the Getting Started guide: https://linuxcontainers.org/lxd/getting-started-cli/#images-for-virtual-machines
I'm not sure why it was added there, but it was part of the images section - I don't feel that is where it belongs though, that's why I moved it to the instance creation.

And my assumption was that using the cloud images is especially important for VMs, because those need the lxd-agent to be able to be able to log on - but that might be wrong. ;)

So do we generally want to recommend using the cloud variants?

Reply:
The `lxd-agent` is included in all VM images we can get it into, so its not specific to having a cloud variant.
Also the concept of a "cloud variant" is only relevant for the "images:" remote and not for the "ubuntu:" remote (as these always include cloud-init and there is no "cloud variant").

I agree it makes sense to keep it in instance creation section.

I agree its usually a good idea to use a VM variant that has cloud-init in it so as to get the root filesystem auto-grow feature.
However I think we should explain a bit more about which images contain cloud-init and why that is recommended for VMs particularly (for the reason above).

Reply:
Also having an example of an image address with a "cloud tag" would be good as its not so clear here currently.

Reply:
Actually, I'm starting to think we should probably leave this change out for now.
From what I understand, we should include the information about the root file system auto-grow feature in the cloud-init docs when we clean them up. And then we can add a hint here about what image to select for a VM and why. But including the full explanation here would be out of place imo.
Does that sound okay? Then I'll make a note of it for cloud-init and remove this change from the PR.

Reply:
Sounds good.

Reply:
OK, updated. :)